.github/workflows/rustdoc.yaml

```diff
@@ -27,10 +27,8 @@ jobs:
       run: cargo doc --all --no-deps
 
     - name: Deploy Docs
-      uses: peaceiris/actions-gh-pages@v2.6.0-rc0
-      env:
-        ACTIONS_DEPLOY_KEY: ${{ secrets.ACTIONS_DEPLOY_KEY }}
-        PUBLISH_BRANCH: gh-pages
-        PUBLISH_DIR: ./target/doc
-      with:
-        forceOrphan: true
+      uses: peaceiris/actions-gh-pages@364c31d33bb99327c77b3a5438a83a357a6729ad # v3.4.0
```

Comment:
@matklad could you please elaborate why this commit sha works, but newer versions don't?

Reply:
Newer versions might work as well, but I don't want to randomly pull unknown new code into CI job which runs with high privileges.

Reply:
Basically, it's a poor-man Cargo.lock